modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -470,19 +472,19 @@ CREATE OR REPLACE TRIGGER IDP_TRIG
                    SELECT IDP_SEQ.nextval INTO :NEW.ID FROM dual;
                END;
 /
-INSERT INTO IDP (TENANT_ID, NAME, HOME_REALM_ID) VALUES (-1234, 'LOCAL', 'localhost')
-/
 CREATE TABLE IDP_ROLE (
-            ID INTEGER,
-            IDP_ID INTEGER,
-            TENANT_ID INTEGER,
-            ROLE VARCHAR(254),
-            PRIMARY KEY (ID),
-            UNIQUE (IDP_ID, ROLE),
-            FOREIGN KEY (IDP_ID) REFERENCES IDP(ID) ON DELETE CASCADE)
+			ID INTEGER,
+			IDP_ID INTEGER,
+			TENANT_ID INTEGER,
+			ROLE VARCHAR(254),
+			PRIMARY KEY (ID),
+			UNIQUE (IDP_ID, ROLE),
+			FOREIGN KEY (IDP_ID) REFERENCES IDP(ID) ON DELETE CASCADE)
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04